include/picongpu/fields/background/templates/twtstight/GetInitialTimeDelay_SI.tpp

```diff
@@ -80,18 +80,18 @@ namespace picongpu
                         /* halfSimSize[2] --> Half-depth of simulation volume (in z); By geometric
                          * projection we calculate the y-distance walkoff of the TWTS-pulse.
                          * The abs( )-function is for correct offset for -phi<-90Deg and +phi>+90Deg. */
-                        float_64 const y1 = float_64(halfSimSize[2] * picongpu::sim.si.getCellSize().z())
-                            * math::abs(math::cos(eta));
+                        float_64 const y1
+                            = float_64(halfSimSize[2] * picongpu::SI::CELL_DEPTH_SI) * math::abs(math::cos(eta));
```

Comment:
1. This happened during a rebase-session.
2. The variable "eta" is only defined and used in this method to guess an initial time delay parameter in case it is not already provided by the user.